lua/graph.lua

```diff
@@ -3,964 +3,887 @@
 --postgres
 
 highway = {
-["motorway"] =          {["auto_forward"] = "true",  ["truck_forward"] = "true",  ["bus_forward"] = "true",  ["taxi_forward"] = "true",  ["moped_forward"] = "false", ["motorcycle_forward"] = "true",  ["pedestrian"] = "false", ["bike_forward"] = "false"},
```

Comment:
What's going on in this file? If it's only formatting, could you move this to a separate PR with an added check in CI to keep it formatted?

Reply:
oh crap, that's terrible, somehow skipped my attention.. sorry, will revert..

Reply:
done